doto-frontend/src/routes/Route.js

```diff
@@ -42,18 +43,12 @@ const Routes = () => {
     return (
         <Switch>
             <Route exact path="/" component={Login} />
-            <Route path="/settings">
-                <ThemeContext.Provider value={[theme, setTheme]}>
-                    <SettingsPage />
-                </ThemeContext.Provider>
-            </Route>
-            <Route path="/calendar">
-                <ThemeContext.Provider value={[theme, setTheme]}>
-                    <Calendar />
-                </ThemeContext.Provider>
-            </Route>
             <Route path="/login" component={Login} />
-            <Route component={NotFound} />
+            <ThemeContext.Provider value={[theme, setTheme]}>
+                <PrivateRoute path="/calendar" component={Calendar}/>
+                <PrivateRoute path="/settings" component={SettingsPage}/>
```

Comment:
Should these have the `exact` prop? Otherwise /settingsXYZ will also go to the settings page instead of 404.

Reply:
Added.